test/test_dblib.ml

```diff
@@ -35,17 +35,27 @@ let test_connect ((_, _, _, database) as params) _ =
 
 let test_basic_query params _ =
   with_conn params (fun conn ->
-      Dblib.sqlexec conn "SELECT CAST(1 AS INT) AS test";
+      Dblib.sqlexec conn
+        "SELECT CAST(1 AS INT) AS test, CAST('' AS VARCHAR(10)) AS str, \
+         CAST(NULL AS BIT) AS n";
```

Comment:
Note: The casts here are just to avoid different DB versions returning these as different kinds of int, str, etc. (And the return type of a NULL seems completely arbitrary).

Reply:
OK.  I think that comment should be in the file itself.